galley/pkg/server/server.go

```diff
@@ -98,11 +102,18 @@ func newServer(a *Args, p patchTable) (*Server, error) {
 	converterCfg := &converter.Config{Mesh: mesh}
 
 	var src runtime.Source
+
+	// Config Source is exclusive
```

Comment:
Why? They should not be - many sources can publish config update events for Galley to process.